tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"allowSyntheticDefaultImports": true,
+		"checkJs": true,
+		"jsx": "preserve",
+		"lib": ["dom", "esnext"],
+		"module": "commonjs",
+		"noEmit": true,
+		"resolveJsonModule": true,
+		"target": "esnext",
+
+		/* Strict Type-Checking Options */
+		"strict": true,                        /* Enable all strict type-checking options. */
+		"noImplicitAny": false,                /* Raise error on expressions and declarations with an implied 'any' type. */
```

Comment:
I tried disabling this and the only error was missing types for `qs` module. Do you expect we'll start pulling in type declarations as `devDependencies`?

I'm on the fence because it seems much easier to relax things when you're too strict than to become more strict when you've been too relaxed 🙂 

Have you already investigated parts of the code where strictness here becomes overly cumbersome and could you provide an example?

Reply:
> I'm nervous that we'd lose people who bought into this crazy idea if I told them we'd have to depend on more external types.

Good point 👍